configure.ac

```diff
@@ -760,13 +760,12 @@ if test "X$BUILD_XDR" != "Xyes"; then
   HAVE_TIRPC="yes"
   ## save current settings
   SYSCPPFLAGS="$CPPFLAGS"
+  SYSCFLAGS="$CFLAGS"
```

Comment:
Without SYSFLAGS being set, the following feature tests clobber CFLAGS and can't restore them